test cases/common/60 install subdir/test.json

```diff
@@ -11,6 +11,7 @@
     {"type": "file", "file": "usr/share/sub1/third.dat"},
     {"type": "file", "file": "usr/share/sub1/sub2/data2.dat"},
     {"type": "file", "file": "usr/share/sub2/one.dat"},
-    {"type": "file", "file": "usr/share/sub2/dircheck/excluded-three.dat"}
+    {"type": "file", "file": "usr/share/sub2/dircheck/excluded-three.dat"},
+    {"type": "file", "file": "usr/share/new_directory"}
```

Comment:
shouldn't this be type "dir" (or "directory", can't remember which)?

Reply:
fixed, thanks.